content/docs/guides/contribute/contribute-to-code/share-your-changes.en.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Share your changes"
+linkTitle: "Share your changes"
+weight: 5
+description: "How to share your code with the project to be reviewed?"
+---
+
+The author of a PR is responsible for its "life". He is responsible for contacting the various parties involved, following the review, responding to comments and correcting the code following review (you could also check [dedicated page about code review]({{< ref "/docs/guides/contribute/code-review" >}})).
+
+> In the case of a large PR, don't hesitate to ask several reviewers to organize themselves, or even to carry out the review together, reviewers and author.
+
+1. **Open a pull request**  
+   Once your changes are ready, you have to request integration with the `dev` branch.
+
+   If possible:
+      - Make PR of logical and atomic units too (avoid mixing refactoring, new features and bug fix at the same time).
+      - Add a description to PRs to explain what they do and why.
+      - Help the reviewer by following advice given in [mtlynch article](https://mtlynch.io/code-review-love/).
+      - Add tags `Area:<affected_area>` to show which part of the application have been impacted. It can be done through [the web interface](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+
+2. **Take feedback into account**  
+   Once your pull request is open, [other contributors can review your changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews):
+
+   - Any user can review your changes.
+   - Your code has to be approved by a contributor [familiar with the code](https://github.com/osrd-project/osrd/blob/dev/.github/CODEOWNERS)
+   - All users are expected to take comments into account.
+   - Comments tend to be written in an open and direct manner.
+     The intent is to efficiently collaborate towards a solution we all agree on.
+   - Once all discussions are resolved, a maintainer integrates the change.
+
+3. **If you believe somebody forgot to review / merge your change, please speak out, multiple times if needs be.**
+
+## Flow proposal
```

Comment:
It won't be a proposal once its merged

Reply:
You're right but ideally I would love to have those two sections side by side but that's not possible.
If I put the flow at first, readers won't read the text, and because I feel that the text is more important — and the flow graph is just the same thing as a graph — I prefer to let this as this.